### DISCUSSION TITLE
I want to make a nodejs version of your code

### DISCUSSION THREAD

**Discussion by Alokkumar8** [acheong08/ChatGPT#32] [Ideas]
Hi, i like your project. I had did similar REST api based reverse engineering before. I want to make a nodejs version of your code and deploy it on vercel to make it free to use for all. Should we collaborate?

**Comment by acheong08** [comment]
Already exists https://github.com/SpecialAgency-Chat/TChatGPT. You can fork it and begin working where they left off.

**Reply by acheong08** [reply]
I am not skilled at JS so you would have better luck working with them

**Reply by Alokkumar8** [reply]
ah ok
